### ISSUE TITLE
Password protecting Mongo

### ISSUE THREAD

**Issue by edemaine** [arunoda/meteor-up-legacy#908]
As I understand it, the Mongo setup made automatically by meteor-up is controlled to accept only local connections.  As a result, any local user can access the database.  I am deploying to a server that has other users, so this protection isn't enough.  Is it possible for meteor-up to automatically set up password-based (or other) authentication for use by Meteor?

**Comment by MasterJames** [comment]
Hmm... I'm not 100% on that. You can set the Mongo url to your own manually (or otherwise) setup instance of MongoDB. 
This might be a good suggestion for consideration in the new version being ported over at kadirahq. The devs are focused and more attentive there, so you could inquire there about it and ask if there's a solution on the original dev branch mupx or not. Basically the newer versions are docker centric and so maybe there's a way that's protected through some docker network isolation magic.
 I'll be thinking more about this later and try to add something if I learn more.

**Comment by edemaine** [comment]
Right, I know I could setup Mongo myself with password authentication; I was hoping that the "setupMongo" feature could do this automatically.  I guess that's not an existing feature, but I could request it on the kadirahq version.  Thanks!
